### PR TITLE
docker.yaml: Provide `BUILD_TAGS` for docker build steps.

### PR DESCRIPTION
**Pull request by hjpotter92**
**What does this pull request do? Explain your changes. (required)**

Adds missing `BUILD_TAGS` arg when building docker image

### CODE REVIEW

**Review by victorges — APPROVED** [review]
LGTM

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -38,13 +54,14 @@ jobs:
...
             type=semver,pattern={{major}}.{{minor}}
-            type=raw,value={{date 'YYYYMMDDHHmmss'}}
             type=raw,value=latest,enable={{is_default_branch}}
             type=raw,value=${{ github.event.pull_request.head.ref }}
 
       - name: Build minimal livepeer distributable
         uses: docker/build-push-action@v3
         with:
+          build-args: |
+            BUILD_TAGS=${{ steps.build-tag.outputs.build-tags }}
```
Comment (victorges) [inline .github/workflows/docker.yaml:61]: Isn't the output name just `tags`?

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -13,6 +13,22 @@ jobs:
...
+          # Check https://github.com/livepeer/go-livepeer/pull/1891
+          # for ref value discussion
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Get build tags
+        id: build-tag
+        run: |
+          ./ci_env.sh echo '::set-output name=build-tags::$BUILD_TAGS'
+          set -x
+          echo "::set-output name=tags::$BUILD_TAGS"
```
Comment (victorges) [inline .github/workflows/docker.yaml]: Actually, they are inconsistent, the first one [1] has `build-tags` while the second one has `tags`.

[1] I actually don't know what that first echo does. What is it for?

File: ci_env.sh [hunk 3]
```diff
@@ -55,4 +55,6 @@ if [[ $generatedVersion != $definedVersion ]]; then
   export BUILD_TAGS="${BUILD_TAGS},experimental"
 fi
 
+echo "::set-output name=build-tags::${BUILD_TAGS}"
+
 exec "$@"
```
Comment (victorges) [inline ci_env.sh:60]: Do we still need this?

**Review by victorges — APPROVED** [review]
LGTM

File: ci_env.sh (hunk 3 above)
Comment (hjpotter92) [reply ci_env.sh:60]: the build script in `build.yaml` still uses this as `./ci_env.sh make` (and it works o_0)

File: docker/Dockerfile [hunk 4]
```diff
@@ -1,68 +1,66 @@
-FROM --platform=$BUILDPLATFORM ubuntu:18.04 as build
```
Comment (iameli) [inline docker/Dockerfile:1]: @hjpotter92 This some kind of autoformatter?

Comment (hjpotter92) [reply docker/Dockerfile:1]: yeah, dockfmt: <https://github.com/jessfraz/dockfmt>

**Pull request merged**